orchestrator/index.js

```diff
@@ -0,0 +1,67 @@
+const request = require('request-promise');
+const HABANERO = require('node-red-habanero');
+
+const HABANERO_ADMIN_ROOT = '/orchestrator';
+const HABANERO_NODE_ROOT  = '/orchestrator-api';
+
+function init(server, app){
+	// Habanero
+	var habaneroSettings = require('../node_modules/node-red-habanero/settings');
```

Comment:
so,  we are pulling in the default node-red settings, and then modifying them here.  Those default settings are not exposed as part of the package. So we reach into the folder and get the defaults this way.  There might be a nicer way of doing this, BUT the issue is the module does expose a `settings` property, but this is not ready until habanero is up and runnning.